pkg/cache/file_based.go

```diff
@@ -19,6 +19,25 @@ func (f *FileBasedCache) IsCacheDisabled() bool {
 	return f.noCache
 }
 
+func (*FileBasedCache) List() ([]string, error) {
```

Comment:
What is the purpose of this function? Apart from knowing the name of the files, I don't see the value it will have for the user. What do you think about it?

Reply:
This is eventually going to be built out to let you delete parts of the cache, for now it's just a utility to see what's in there